### PR TITLE
update and queries

### PR DESCRIPTION
**Pull request by Arturgouveia1970**
In this PR, I updated the database and made queries to the DB.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -5,4 +5,4 @@ CREATE TABLE animals (
     escape_attemps integer,
```
Comment (Danieldotcomcoder) [inline schema.sql:5]: Inside the schema file, you should add a column named species to the animals table as mentioned in the project requirements 👍😀
kindly fix this issue👍.

File: data.sql [hunk 2]
```diff
@@ -11,5 +11,21 @@ INSERT INTO animals ( id, name, date_of_birth, escape_attemps, neuterd, weight_k
...
+ ADD AUTO_INCREMENT;
+ INSERT INTO animals (name, date_of_birth, escape_attemps, neuterd, weight_kg) values ('bobby', '2020-04-04', 1, true, 10);
+ALTER TABLE animals 
+ALTER COLUMN id ADD GENERATED ALWAYS AS IDENTITY;
+delete from animals where id = 21;
+update animals set species = 'digimon' where name like '%mon';
+update animals set species = 'pokemon' where species is null;
+delete from animals where date_of_birth > '2022-01-01';
+SAVEPOINT younger_than_2022_animals;
+ROLLBACK;
```
Comment (Danieldotcomcoder) [inline data.sql:28]: kindly here you should update animal weights as mentioned in the requirements 😀👍

image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Arturgouveia1970, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Arturgouveia1970 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**